downloads.html

```diff
@@ -13,6 +13,37 @@ <h1>Download</h1>
         
         <h2>Supported releases</h2>
 
+        <span id="4.0.0"></span>
+        <h3 class="download-version">4.0.0<a href="#4.0.0"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+        <ul>
+            <li>
+                Released March 18, 2025
+            </li>
+            <li>
+                <a href="https://dlcdn.apache.org/kafka/4.0.0/RELEASE_NOTES.html">Release Notes</a>
+            </li>
+            <li>
+                Docker image: <a href="https://hub.docker.com/layers/apache/kafka/4.0.0/images/sha256-01b9a4030e54c6068e66eb3ba4cb82c0d89238629ef1c30d79b86036bf89b1b7">apache/kafka:4.0.0</a>.
+            </li>
+            <li>
+                Docker Native image: <a href="https://hub.docker.com/layers/apache/kafka-native/4.0.0/images/sha256-94061e3c94f33ae18011e0f4e96c03a6ec4def97bbbad41ff68473e0a04c4ad8">apache/kafka-native:4.0.0</a>.
+            </li>
+            <li>
+                <!-- Note that source/binary downloads should point to `dlcdn.apache.org` but checksum files must point to `downloads.apache.org` -->
+                Source download: <a href="https://dlcdn.apache.org/kafka/4.0.0/kafka-4.0.0-src.tgz">kafka-4.0.0-src.tgz</a> (<a href="https://downloads.apache.org/kafka/4.0.0/kafka-4.0.0-src.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/4.0.0/kafka-4.0.0-src.tgz.sha512">sha512</a>)
+            </li>
+            <li>
+                <!-- Note that source/binary downloads should point to `dlcdn.apache.org` but checksum files must point to `downloads.apache.org` -->
+                Binary downloads: <a href="https://dlcdn.apache.org/kafka/4.0.0/kafka_2.13-4.0.0.tgz">kafka_2.13-4.0.0.tgz</a> (<a href="https://downloads.apache.org/kafka/4.0.0/kafka_2.13-4.0.0.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/4.0.0/kafka_2.13-4.0.0.tgz.sha512">sha512</a>)
```

Comment:
Let's file a JIRA to remove the `_2.13` from the distribution name. It doesn't seem useful when we only support one Scala version.

Reply:
https://issues.apache.org/jira/browse/KAFKA-12622